firebase/functions/index.js

```diff
@@ -163,6 +130,21 @@ exports.counter = functions.database.ref('/v2/results/{projectId}/{groupId}/{use
 })
 
 
+exports.counter_2 = functions.database.ref('/v2/users/{userId}/contributions/{projectId}/').onCreate((snapshot, context) => {
```

Comment:
I like the approach to put this into a separate function.

Maybe you can find a better name than `counter_2`. :)  e.g. `projectCounter` or something similar.

Reply:
Yeah counter_2 is not an elegant or meaningful name :D
I will think of something better and as you said include some more comments and maybe restructure more code if that improves readability. It took me some time to figure out again which attribute / db ref is depended on which other and what will execute when.